### ISSUE TITLE
URL STATUS

### ISSUE THREAD

**Issue by sathiyarajcbe** [tonyofrancis/Fetch#115]
Hello how can i get the requested url status? file exists or not ...!

**Comment by tonyofrancis** [comment]
@sathiyarajcbe I do not understand your question. Can you rephrase it?

**Comment by sathiyarajcbe** [comment]
If we are submitting download request but if the file is not available in the requested URL means we are experiencing the app crashes...

**Comment by tonyofrancis** [comment]
@sathiyarajcbe Can you share the crash logs. I will have to replicate this. Thanks

**Comment by anhtuannd** [comment]
I experience the same problem. In my opinion, we should call FetchListener::onError.
